style.css

```diff
@@ -23,13 +23,41 @@ header {
   padding-top: 28px;
 }
 
+.nav-menu {
+  display: none;
+}
+
+.navbar {
+  display: flex;
+}
+
+.humburger {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding-right: 6px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+#openBtn {
+  margin-left: 380px;
+}
```

Comment:
- [OPTIONAL] Kindly remove this margin and position this button differently, making it close to the top right corner like the Figma template. This button is in the correct position, but due to a fixed and large margin, on small mobile screens this button doesn't show up for the user making the mobile menu impossible to reach.